composer.json

```diff
@@ -7,6 +7,10 @@
         {
             "name": "Taylor Otwell",
             "email": "taylorotwell@gmail.com"
+        },
+        {
+            "name": "Tom de Goede",
+            "email": "tom@allesonline.eu"
```

Comment:
I have no idea how that ends up here, been messing with satis and composer etc. Was not intended to end up here.